packages/liveblocks-node/src/webhooks.ts

```diff
@@ -27,13 +27,18 @@ export class WebhookHandler {
    * Verifies a webhook request and returns the event
    */
   public verifyRequest(request: WebhookRequest): WebhookEvent {
-    const { webhookId, timestamp, rawSignatures } = this.verifyHeaders(
-      request.headers
-    );
+    const { headers, rawBody } = request;
+
+    const { webhookId, timestamp, rawSignatures } = this.verifyHeaders(headers);
+
+    if (typeof rawBody !== "string") {
+      throw new Error(`Invalid body, must be a string, got "${typeof rawBody}" instead. 
+      It is likely that you need to JSON.stringify the body before passing it.`);
```

Comment:
1. In error messages, prefer not using `\n` since they can get cropped weirdly depending on where they display. Just using a space should be enough.
2. Maybe say which field this is checking in the error message, i.e. say "the `rawBody` field must be …".

Reply:
Thanks, will adjust!